db/internal_stats.h

```diff
@@ -237,6 +237,27 @@ class InternalStats {
       }
     }
 
+	CompactionStats & operator=(const CompactionStats& c) {
```

Comment:
We use space only for indenting, and there is tab here, which is not allowed. Please fix it.

Reply:
Fixed. Travis job glitched and needs to be re-run.